public/app/routes/routes.tsx

```diff
@@ -363,6 +363,12 @@ export function getAppRoutes(): RouteDescriptor[] {
         () => import(/* webpackChunkName: "AdminEditOrgPage" */ 'app/features/admin/AdminEditOrgPage')
       ),
     },
+    {
+      path: '/admin/featuretoggles',
+      component: SafeDynamicImport(
+        () => import(/* webpackChunkName: "AdminFeatureTogglesPage" */ 'app/features/admin/AdminFeatureTogglesPage')
+      ),
+    },
```

Comment:
guard this with a check for the feature toggle

Reply:
thanks for the review, I addressed this

Reply:
I also added the RTK changes, let me know if you have any feedback! tysm